doc/dev/crimson.rst

```diff
@@ -0,0 +1,143 @@
+=======
+crimson
+=======
+
+crimson is the code name of crimson-osd, which the next generation ceph-osd. It
+targets fast storage devices, like NVMe storage, for better performance.
```

Comment:
Currently the scope is (block-addressable) NVMe SSD and (byte-addressable) persistent memory, right?

Reply:
i think we will also support HDD, and none NVMe SSD.